ui/raidboss/data/02-arr/dungeon/cutters_cry.ts

```diff
@@ -0,0 +1,29 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { TriggerSet } from '../../../../../types/trigger';
+
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.CuttersCry,
+  triggers: [
+    {
+      id: 'Chimera Ram Voice',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '450', source: 'Chimera', capture: false }),
+      suppressSeconds: 0.1,
```

Comment:
Are there really multiple starts using lines here that need to be suppressed?

Reply:
Oops, nope.